swarm/metrics/flags.go

```diff
@@ -91,7 +92,10 @@ func Setup(ctx *cli.Context) {
 		)
 
 		// Start system runtime metrics collection
-		go gethmetrics.CollectProcessMetrics(2 * time.Second)
+		go gethmetrics.CollectProcessMetrics(4 * time.Second)
```

Comment:
you specified 4, not 10 as you wrote in the comment

Reply:
10sec. is the resolution (a few lines below), because this is how often the process sends out the metrics to InfluxDB.

So basically we collect them twice in process, but send them out once.